### PR TITLE
syscontainers: allow --system without runc installed

### PR DESCRIPTION
**Pull request by giuseppe**
Also change --user to work without bwrap-oci being installed

Closes: https://github.com/projectatomic/atomic/issues/866

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: Atomic/install.py [hunk 1]
```diff
@@ -49,17 +49,12 @@ def cli(subparser):
...
-            if bwrap_oci_available:
-                system_xor_user.add_argument("--user", dest="user", action="store_true", default=False,
-                                             help=_("Flag to specify if user is non-root privileged."))
-            if runc_available:
-                system_xor_user.add_argument("--system", dest="system",
-                                             action='store_true', default=False,
-                                             help=_('install a system container'))
+        system_xor_user = installp.add_mutually_exclusive_group()
+        system_xor_user.add_argument("--user", dest="user", action="store_true", default=False,
+                                     help=_("Flag to specify if user is non-root privileged."))
```
Comment (rhatdan) [inline Atomic/install.py]: Shouldn't this be soemthing like

Install the image as a user image?  Or a non-root image?

Comment (rhatdan) [inline Atomic/install.py]: Flag should not be capitalized either.

**Comment by miabbott** [comment]
Looks good from here

```
# git rev-parse HEAD
b36ab020d7635c6a3c5ab0b1987948537504ab9b
# rpm -q ostree runc
ostree-2017.1-2.fc25.x86_64
package runc is not installed
# ./atomic images list -a
# ./atomic install --system gscrivano/etcd
Pulling layer 0fc456f626d7a9b48198aae17fdcfc3881b94528b052fa5e0d5a64ac5b16e4d6
Pulling layer b9cb8f274ab4f83ddc52faef044c7af112b16df6dd2c53352284f238e83309fa
Pulling layer 526d753d265db3ce10564b02eaccae0a12aaad9c0de68cfe16a3be7b89ccf099
Pulling layer 26d593e5c67936fdc4a0f12a7895f1194b0ba9de2f0c6c7804e1f600195ddc34
Pulling layer b09724a96f1b6c2d9d8a9b894870fd71703b73797c80999f5bd00a2a16b8d4ce
Pulling layer ac23bb9fe05b79b47bce2157235bc751221100bdb49abd3c17d1a530e0aabe62
Extracting to /var/lib/containers/atomic/etcd.0
systemctl daemon-reload
systemd-tmpfiles --create /etc/tmpfiles.d/etcd.conf
systemctl enable etcd
```

File: Atomic/install.py (hunk 1 above)
Comment (giuseppe) [inline Atomic/install.py]: I've changed it in a fixup patch

File: Atomic/util.py [hunk 2]
```diff
@@ -40,9 +40,6 @@ def gomtree_available():
 def runc_available():
     return os.path.exists(RUNC_PATH)
 
-def bwrap_oci_available():
```
Comment (giuseppe) [inline Atomic/util.py]: we don't use it anywhere now.

bwrap-oci should not be required for user images, in the same way as we don't require runc for system containers.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ -r rhatdan b36ab02

**Comment by giuseppe** [comment]
rh-atomic-bot r=rhatdan b36ab02

**Comment by giuseppe** [comment]
@rh-atomic-bot retry

**Comment by rhatdan** [comment]
@rh-atomic-bot r=rhatdan b36ab02

**Comment by rhatdan** [comment]
@rh-atomic-bot r=rhatdan 684b2fa

**Pull request closed without merging**